cmd/istioctl/main.go

```diff
@@ -163,11 +178,11 @@ var (
 					c.Println(c.UsageString())
 					return err
 				}
-				item, exists := config.Get(key)
-				if !exists {
-					return fmt.Errorf("%q does not exist", key)
+				config, err := apiClient.GetConfig(key)
+				if err != nil {
+					return err
 				}
-				out, err := schema.ToYAML(item)
+				out, err := schema.ToYAML(config.ParsedSpec)
```

Comment:
This was here previously, I assumed it was to verify what we retrieved was actually a legitimate entry.